.github/workflows/prereleaser.yaml

```
@@ -18,9 +18,43 @@
+  verify:
+    runs-on: ubuntu-latest
+    steps:
+      - id: token
+        name: github-token-gen
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.PRIMUS_APP_ID }}
+          private-key: ${{ secrets.PRIMUS_PRIVATE_KEY }}
+          owner: ${{ github.repository_owner }}
+      - name: verify-user
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2036:warning:13:1: If you wanted to assign the output of the pipeline, use a=$(b | c) 

(shellcheck)